_pages/introduction-developing-skills.md

```diff
@@ -158,13 +158,13 @@ For example, how do you say 'goodbye' to someone?
 
 #### vocab directory and defining Intents
 
-Each **Skill** defines one or more **Intents**. Intents are defined in the 'vocab' directory. The 'vocab' directory is organized by language, just like the 'dialog' directory.  
```

Comment:
I'm curious why this line has been removed; I think this line is needed to make it clear that both the `vocab` and `dialog` directories are organized by language.